pkg/security/config/config.go

```diff
@@ -90,17 +90,19 @@ type Config struct {
 	SelfTestEnabled bool
 	// EnableRemoteConfig defines if configuration should be fetched from the backend
 	EnableRemoteConfig bool
-	// EnableRuntimeCompiledConstants defines if the runtime compilation based constant fetcher is enabled
-	EnableRuntimeCompiledConstants bool
-	// RuntimeCompiledConstantsIsSet is set if the runtime compiled constants option is user-set
-	RuntimeCompiledConstantsIsSet bool
 	// ActivityDumpEnabled defines if the activity dump manager should be enabled
 	ActivityDumpEnabled bool
 	// ActivityDumpCleanupPeriod defines the period at which the activity dump manager should perform its cleanup
 	// operation.
 	ActivityDumpCleanupPeriod time.Duration
 	// RuntimeMonitor defines if the runtime monitor should be enabled
 	RuntimeMonitor bool
+	// RuntimeCompilationEnabled defines if the runtime-compilation is enabled
+	RuntimeCompilationEnabled bool
```

Comment:
Is there a reason to have a separate config option for CWS? If so, it might need to respect the system-probe level option, if it is set and explicitly set to false.

Reply:
The reason is to have a more fine grained runtime-compilation configuration. I'll add something to handle the case where at the system-probe level the runtime-compilation is disabled